jellyfin-api/src/jvmMain/kotlin/org/jellyfin/sdk/api/client/KtorClient.kt

```diff
@@ -121,4 +105,10 @@ public actual open class KtorClient actual constructor(
 			throw err
 		}
 	}
+
+	private fun HttpMethod.asKtorHttpMethod(): io.ktor.http.HttpMethod = when (this) {
```

Comment:
Maybe do an alias-import on `io.ktor.http.HttpMethod` as `KtorHttpMethod`? Because this is rather ugly..